ui/src/pages/TargetCreate.tsx

```diff
@@ -21,7 +21,9 @@ const TargetCreate = () => {
           if (connStatus === ExternalConnectivityStatus.TLSConfirmFingerprint) {
             notify.error(`Successfully added new target ${values.name}, but received an untrusted TLS server certificate with fingerprint ${response.metadata?.["certFingerprint"]}. Please update the source to correct the issue.`);
           } else if (connStatus === ExternalConnectivityStatus.WaitingOIDC) {
-            notify.error(`"Successfully added new target ${values.name}; please visit ${response.metadata?.["OIDCURL"]} to complete OIDC authorization."`);
+            const oidcURL = response.metadata?.["OIDCURL"];
+            notify.error(`"Successfully added new target ${values.name}; please visit ${oidcURL} to complete OIDC authorization."`);
```

Comment:
Should that be notified as an error though?
Having an error message start with `Successfully` feels pretty odd :)

Given we auto-open the tab, maybe make it an info message.

Possibly changing to something like this:
```
Successfully added new target. Please go to <LINK> if your browser didn't open an authentication window for you.
```

Reply:
@stgraber I agree with you. Actualy this was based on what we have in cli. Where this special cases are handled as errors: https://github.com/FuturFusion/migration-manager/blob/216d8d824b48b4ef00beec7241ec2b037b1c3e04/cmd/migration-manager/internal/cmds/target.go#L455-L461

I'll improve UI part.